cmd/fleet/cron.go

```diff
@@ -838,6 +832,50 @@ func newCleanupsAndAggregationSchedule(
 	return s, nil
 }
 
+func newFrequentCleanupsSchedule(
+	ctx context.Context,
+	instanceID string,
+	ds fleet.Datastore,
+	lq fleet.LiveQueryStore,
+	logger kitlog.Logger,
+) (*schedule.Schedule, error) {
+	const (
+		name            = string(fleet.CronFrequentCleanups)
+		defaultInterval = 15 * time.Minute
+	)
+	s := schedule.New(
+		ctx, name, instanceID, defaultInterval, ds, ds,
+		// Using leader for the lock to be backwards compatilibity with old deployments.
+		schedule.WithAltLockID("leader"),
```

Comment:
Using this will collide with the `schedule.WithAltLockID("leader")` used in `newCleanupsAndAggregationSchedule`.